modules/scalacheck-toolbox-datetime/src/main/scalajvm/com/fortysevendeg/scalacheck/datetime/joda/GenJoda.scala

```diff
@@ -23,15 +23,16 @@ import scala.util.Try
 import org.scalacheck.Gen
 import org.scalacheck.Arbitrary
 import org.joda.time._
+import com.fortysevendeg.scalacheck.datetime.YearRange
 
 /**
  * Generators specific for Joda time.
  */
 trait GenJoda {
```

Comment:
Random question, unrelated to your PR. Can't we remove support for joda-time? Does anyone still use that?